content/en/truecommand/overview/tcwhatis.md

```diff
@@ -4,9 +4,51 @@ description: "Describing the features and some use cases of the TrueCommand NAS
 weight: 10
 ---
 
-TrueCommand is a single Pane-of-Glass management application that takes the repetitive work out of multi-TrueNAS management by centralizing system alerts, reports, and analytics in one easy to use interface.
-It supports users and teams with 24x365 global operations, role-based access control, and a full log of every TrueNAS configuration change.
-TrueCommand runs on docker, VMs, or as a cloud service and is free to users with less than 50 drives and affordable for those with larger installations.
+### What is TrueCommand?
 
-<img src="/images/truecommand-overview.png">
+TrueCommand is a multi-system management "Single pane of Glass" system that helps control and monitor your TrueNAS fleet. TrueCommand assists in managing TrueNAS systems through REST APIs, WebSocket APIs, and a web user interface. The TrueCommand web interface provides unified administration of users and TrueNAS systems and single sign-on functionality.
+
+TrueCommand can monitor an entire fleet of TrueNAS systems and thousands of online storage devices simultaneously. This includes displaying statistics on storage usage, network activity, active services, and more. TrueCommand also has the ability to create custom reports about individual systems or a combination of many systems.
+
+<img src="/images/truecommand-overview.png" width='700px'>
 <br><br>
+
+### What features does TrueCommand have?
+
+- Multiple deployment options: TrueCommand is a lightweight application that supports deployments in either a Virtual Machine or a Docker Container. TrueCommand Cloud is also available as a cloud-based subscription option that allows you to offload TrueCommand resources and deployment and only focus on fine-tuning your configuration.
+
+- NAS Fleet Dashboard: The TrueCommand dashboard provides visibility to an organization’s entire TrueNAS fleet. TrueCommand includes an auto-discovery tool that expedites identifying and integrating systems into TrueCommand.
+
+- Single Sign-on to all NAS Units
+ Once a TrueNAS system has been connected to TrueCommand, authorized administrators can quickly log in to that system through TrueCommand. This allows for quicker and simpler signons instead of looking up IP addresses and login credentials.  This is even more beneficial when using different secure passwords for each TrueNAS instance instead of a single password across multiple systems.
```

Comment:
Should "TrueCommand, authorized"  be "TrueCommand, and authorized"

Reply:
Not in this instance.  English grammar can be weird.  "Once a TrueNAS system has been connected to TrueCommand" is an introductory clause, so 'and' would be wrong.   But for ease of translation later I probably should re-write that sentence to be simpler.

Reply:
The new version is way better.